lib/whitehall_importer/integrity_checker.rb

```diff
@@ -138,5 +113,55 @@ def default_image?(proposed_image_payload, publishing_api_image, attribute)
         proposed_image_payload.empty? &&
         publishing_api_image[attribute] == "placeholder"
     end
+
+    class ContentIntegrityChecker < IntegrityChecker
```

Comment:
Suggestion that we move this into an integirty_checker directory

Also that we only deal with the body in here, then we can avoid the need to subclass and be simpler to test.

Reply:
Good idea.  Have moved this and also broken the commit down into two (one to move the code, the other to add the file size ignoring).